apps/emqx_durable_storage/src/emqx_ds_lts.erl

```diff
@@ -115,6 +120,10 @@ trie_create() ->
 -spec trie_restore(options(), [{_Key, _Val}]) -> trie().
 trie_restore(Options, Dump) ->
     Trie = trie_create(Options),
+    trie_restore_existing(Trie, Dump).
+
+-spec trie_restore_existing(trie(), [{_Key, _Val}]) -> trie().
+trie_restore_existing(Trie, Dump) ->
     lists:foreach(
         fun({{StateFrom, Token}, StateTo}) ->
```

Comment:
I think this function must ignore all but wildcard routes. Otherwise it will copy the trie elements created during the learning; we want to drop them to optimize the number of streams.

Reply:
Makes sense.  But is there a way to know if a given trie dump entry is part of a larger wildcard route without reconstructing the whole trie?  i.e.: just looking at the `Token`?

e.g.: For the filter `t/+/a`, we still need to insert the static `t` and `a`.

Reply:
True. I mean the algorithm should, of course, copy the full paths, but only as long as there is at least one wildcard in it. I think we can postpone the implementation for later, though.